python/python/glide/async_commands/cluster_commands.py

```diff
@@ -555,6 +556,48 @@ async def fcall_ro_route(
             await self._execute_command(RequestType.FCallReadOnly, args, route),
         )
 
+    async def function_stats(
+        self, route: Optional[Route] = None
+    ) -> TClusterResponse[TFunctionStatsResponse]:
+        """
+        Returns information about the function that's currently running and information about the
+        available execution engines.
+
+        See https://redis.io/commands/function-stats/ for more details
```

Comment:
Are we sure we want to mix use `redis.io` and `valkey.io`? Most python commands use `valkey.io` though.